src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Sudoku {
+    public Sudoku(Scanner gameScan) {
+        this.gameScan = gameScan;
+    }
+
+    private  final Scanner gameScan;
+
+    public  boolean startup() {
+
+        System.out.println("Witamy w Sudoku Zugaj Edition V1\nBy rozpocząć rozgrywkę wpisz:  Sudoku");
```

Comment:
Mieszasz logikę z wyświetlaniem danych. Pokażę ci w środę jak tego uniknąć bo to błąd KARDYNLNY